### DISCUSSION TITLE
text-shadow for button

### DISCUSSION THREAD

**Discussion by devopg** [tailwindlabs/tailwindcss#7776] [Help]
hello how i can add text shadow for btn? 

Tailwind recomended use drop-shadow shadow-color-x but its dont work.
Expected result : text and btn has shadow
![image](https://user-images.githubusercontent.com/61819835/157054179-fe502f36-6edf-4946-9fd9-ebdd93dec655.png)

sandbox:
https://play.tailwindcss.com/tuWYQjGRF0

**Comment by Oluwafemi21** [comment]
Use the shadow utility class. You can also check the docs
[Shadows](https://tailwindcss.com/docs/box-shadow)

**Reply by devopg** [reply]
@Oluwafemi21 
https://tailwindcss.com/docs/box-shadow
its work only for "border shadow" then apply to button. dont give effect for text.
